esp-hal/src/rom/mod.rs

```diff
@@ -62,6 +62,8 @@ macro_rules! regi2c_write {
     };
 }
 
+pub use regi2c_write; // TODO: can be removed as soon as ROM is stabilized
```

Comment:
What if we remove `#[doc(hidden)]` and `#[macro_export]`? We can keep these macros private, and you could remove the TODO.